de-lib/src/main/java/org/iplantc/de/client/models/collaborators/Subject.java

```diff
@@ -73,7 +73,7 @@ default String getSubjectDisplayName() {
         if (Strings.isNullOrEmpty(subjectName)) {
             subjectName = getFirstName() + " " + getLastName();
         }
-        if (!isGroup() || !hasGroupLongName(subjectName)) {
+        if (!isGroup() && !hasGroupLongName(subjectName)) {
             return subjectName;
```

Comment:
Yeah, I need to clean this up - it is absolutely confusing.  This was my lazy way of fixing an issue I had and it makes no sense but it works and then I moved on to the next issue 😄 . 
I'm slightly tempted to ditch the `default` method idea and create a more robust util class that can handle the differences between a `Group` and a `Subject`, but I'm not sure yet.  I'll definitely refactor though.